### PR TITLE
Release 0.1.2 admission controller helm charts

### PR DESCRIPTION
**Pull request by michael-zhou-lw**

### CODE REVIEW

File: admission-controller/values.yaml [hunk 1]
```diff
@@ -8,6 +8,9 @@ admission:
   tls_port: 8443
   cert_file_path: "/certs/admission.crt"
   cert_key_path: "/certs/admission.key"
+  excluded_resources:
+  -
+  -
```
Comment (cirego) [inline admission-controller/values.yaml:13]: You might consider adding a `values.schema.json` to ensure that `excluded_resources` and other Helm chart values are well formed. It also helps to document the expected structure of the arguments (list vs hashmap, etc).

Comment (michael-zhou-lw) [reply admission-controller/values.yaml:13]: Great suggestion! Added this file.

File: admission-controller/values.schema.json [hunk 2]
```diff
@@ -0,0 +1,1002 @@
+{
```
Comment (cirego) [inline admission-controller/values.schema.json:1]: Did you generate this manually or using a tool? I don't see a `$schema` field, so I'm curious as to how this was created. Looks great!

Comment (michael-zhou-lw) [reply admission-controller/values.schema.json:1]: I used a tool and removed it and the linting seemed to work still. Is that field necessary?

Comment (cirego) [reply admission-controller/values.schema.json:1]: It's used to specify the schema version of the JSON file itself. I'd recommend keeping it.

What tool did you use?

Comment (michael-zhou-lw) [reply admission-controller/values.schema.json:1]: Ah that makes sense, I'll add it back in. I used a yaml to json tool and then https://www.jsonschema.net/home.

**Review by ChitraPKumar — APPROVED** [review]
Would double check there is no proxy scanner release this time. 
Thanks Michael.

File: admission-controller/values.schema.json [hunk 3]
```diff
@@ -0,0 +1,1004 @@
...
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "http://example.com/example.json",
```
Comment (cirego) [inline admission-controller/values.schema.json:3]: Hi @michael-zhou-lw, the `$id` field here is supposed to identify this schema uniquely. Perhaps something like `https://lacework.github.io/helm-charts/admission-controller/values.schema.json`?

Comment (michael-zhou-lw) [reply admission-controller/values.schema.json:3]: @cirego I've created another PR to fix this and update the README, could you have a look please? https://github.com/lacework/helm-charts/pull/9

**Pull request merged**